.circleci/config.yml

```diff
@@ -14,25 +14,25 @@ workflows:
       - kitchen/dokken-single:
           name: centos-7
           platform: centos-7
-          suite: default
+          suite: install
```

Comment:
If this is the default set of tests, we can probably just use default for simplicity.

"Hey this is the default setup, and it's mostly covered in this suite"